src/Marten/Patching/PatchOperation.cs

```diff
@@ -30,10 +30,21 @@ public class PatchOperation : IStorageOperation, NoDataReturnedCall
         }
 
         // TODO -- come back and do this with a single command!
+        private const string VALUE_LOOKUP = "___VALUE___";
         public void ConfigureCommand(CommandBuilder builder)
         {
-            var patchJson = _serializer.ToCleanJson(_patch);
-            var patchParam = builder.AddJsonParameter(patchJson);
+            var value = _serializer.ToJson(_patch["value"]);
```

Comment:
Ugh, ugh, ugh. I don't have any other recommendations, but this is making it way more inefficient. Let's let this go for now, but maybe if we ever get around to the Linq overhaul, we can do something with the `IField` model so that it "knows" whether or not it needs to do the heavyweight serialization state here or can get away with something lighter like just the text.